features/shop/cart_with_different_taxes_for _different_zones.feature

```diff
@@ -0,0 +1,48 @@
+@ui-cart
+Feature: Cart with different tax rates for different time zones
+    In order to order products with different tax rate for different zones
+    As a Customer
+    I want to be aware of taxes applied on my order
+
+    Background:
+        Given that store is operating on the France channel
+        And default currency is "EUR"
+        And tax rate "EU VAT" with 23% rate belongs to "Taxable Goods" category for "EU" zone
```

Comment:
:+1: I was wondering about sth conciser, but didn't know how short can it be :smile: And :+1: for more specific category